stable/grafana/values.yaml

```diff
@@ -51,6 +51,7 @@ livenessProbe:
 image:
   repository: grafana/grafana
   tag: 7.0.5
+  sha: 17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
```

Comment:
Are you sure about this sha? https://hub.docker.com/layers/grafana/grafana/7.0.5/images/sha256-ab0ab1ca855b7631588f01e350b850dfb83821ed1865e04e031ba2622ac3a453?context=explore

Reply:
@GMartinez-Sisti could you please take a look?

Reply:
@pierluigilenoci this has reverted default sha https://github.com/helm/charts/pull/23240